src/firebase/ranking.js

```diff
@@ -0,0 +1,82 @@
+import { collection, getDocs, query, orderBy, where } from 'firebase/firestore';
+
+import { GAME_STATUS, RANKING_VALUES } from 'constants/types';
+import { DAILY_RESULTS, USERS_STATISTICS } from 'firebase/collections';
+import firebaseData from 'firebase/firebase';
+
+const { firebaseDb } = firebaseData;
```

Comment:
```suggestion
import { firebaseDb } from 'firebase/firebase';
```
Maybe?

Reply:
Is not working 😢   Because in firebase I export default `firebaseData` that returns `{ firebaseDb }`. 

<img width="551" alt="image" src="https://user-images.githubusercontent.com/8755889/221601092-32b5b022-4a3d-4d3a-8629-e35bc7cbff9d.png">